var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -35,6 +36,7 @@ class Ghostscript(AutotoolsPackage):
     depends_on('gtkplus')
 
     patch('nogoto.patch', when='%fj@:4.1.0')
+    patch('freetype-2.10.4.patch', when='^freetype@2.10.4:')
```

Comment:
Can you add a brief comment on top of the directive that explains what the patch does and why it's needed? Also, if it's easy to retrieve it from a url that would be better.